### PR TITLE
feat: next/font package implementation

### PR DESCRIPTION
**Pull request by alemesa**
* Attempting to use `next/font`
* Slightly better LCP
* Easier to setup than copying fonts like a barbarian

NextJS: https://nextjs.org/docs/pages/building-your-application/optimizing/fonts
NPM package: https://www.npmjs.com/package/@next/font

### CODE REVIEW

**Comment by edgardz** [comment]
@alemesa I like this. What is missing here?

**Comment by alemesa** [comment]
> @alemesa I like this. What is missing here?

honestly nothing @edgardz was gonna show it in the open source meeting, to see what people think, but we don't have open source meeting anymore 💀

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -46,9 +47,11 @@ const App: FC<AppProps<PageProps>> = (props) => {
...
 
   return (
-    <Provider store={store}>
-      {props.pageProps.noLayout ? <props.Component {...props.pageProps} /> : <Layout {...props} />}
-    </Provider>
+    <div className={fontVariables}>
+      <Provider store={store}>
+        {props.pageProps.noLayout ? <props.Component {...props.pageProps} /> : <Layout {...props} />}
+      </Provider>
+    </div>
```
Comment (edgardz) [inline src/pages/_app.tsx:54]: I'm not sure about wrapping everything into a div

Comment (alemesa) [reply src/pages/_app.tsx:54]: Let's do it in Layout then, I'll push a commit

**Review by edgardz — DISMISSED** [review]

**Review by edgardz — APPROVED** [review]

**Pull request merged**